fastlane/Fastfile

```diff
@@ -426,11 +432,12 @@ end
       archive_zip_path = File.dirname(Dir.pwd) + "/Simplenote.xarchive.zip"
       zip(path: lane_context[SharedValues::XCODEBUILD_ARCHIVE], output_path: archive_zip_path)
 
-      version = ios_get_app_version()
+      version = options[:beta_release] ? ios_get_build_version : ios_get_app_version
       create_release(repository:GHHELPER_REPO,
         version: version,
-        release_notes_file_path:'./Simplenote/Resources/release_notes.txt',
-        release_assets:"#{archive_zip_path}"
+        release_notes_file_path: './Simplenote/Resources/release_notes.txt',
```

Comment:
💄 nit: likewise, while we're at it, might as well use the proper `File.join` API to construct the path
```suggestion
        release_notes_file_path: File.join(PROJECT_ROOT_FOLDER, 'Simplenote', 'Resources', 'release_notes.txt'),
```